.jenkins/caffe2/bench.sh

```diff
@@ -13,11 +13,14 @@ else
     num_gpus=0
 fi
 
-cmd="$PYTHON $CAFFE2_PYPATH/python/examples/resnet50_trainer.py --train_data null --batch_size 64 --epoch_size 6400 --num_epochs 2"
 if (( $num_gpus == 0 )); then
-    cmd="$cmd --use_cpu"
+    "$PYTHON" "$CAFFE2_PYPATH/python/examples/resnet50_trainer.py" --train_data null --batch_size 64 --epoch_size 6400 --num_epochs 2 --use_cpu
 else
-    cmd="$cmd --num_gpus 1"
+    "$PYTHON" "$CAFFE2_PYPATH/python/examples/resnet50_trainer.py" --train_data null --batch_size 64 --epoch_size 6400 --num_epochs 2 --num_gpus 1
+    if (( $num_gpus >= 2 )); then
+        "$PYTHON" "$CAFFE2_PYPATH/python/examples/resnet50_trainer.py" --train_data null --batch_size 128 --epoch_size 12800 --num_epochs 2 --num_gpus 2
+    fi
+    if (( $num_gpus >= 4 )); then
```

Comment:
Do we want to test both 2 and 4 on the machines with 4+ gpus?

Reply:
Why not? :-)